content/en/remote_configuration/_index.md

```diff
@@ -0,0 +1,173 @@
+- link: "https://www.datadoghq.com/blog/remote-configuration-for-datadog/"
+  tag: "Blog"
+  text: "Apply real-time updates to Datadog components with Remote Configuration"
+algolia:
+  tags: ['remote config', 'remote configuration']
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Remote Configuration is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+## Overview
+
+Remote Configuration is a Datadog capability that allows you to remotely configure and change the behavior of select product features in Datadog components such as Agents, tracing libraries, and Observability Pipelines Workers deployed in your infrastructure. Use Remote Configuration to apply configurations to Datadog components in your environment on demand, decreasing management costs, reducing friction between teams, and accelerating issue resolution times.
+
+For Datadog security products, App and API Protection and Workload Protection, Remote Configuration-enabled Agents and compatible tracing libraries provide real-time security updates and responses, enhancing security posture for your applications and cloud infrastructure.
+
+## How it works
+
+When Remote Configuration is enabled, Datadog components such as the Datadog Agent securely poll the configured [Datadog site][1] for configuration changes that are ready to apply. Pending changes are then automatically applied to Datadog components. For example, after you submit configuration changes in the Datadog UI for a Remote Configuration-enabled product feature, the changes are stored in Datadog.
+
+The following diagram illustrates how Remote Configuration works:
+
+{{<img src="agent/remote_config/RC_Diagram_v5.png" alt="Users configure features in the UI, the config is stored in Datadog, the Agent requests config updates." width="90%" style="center">}}
+
+1. You configure select product features in the Datadog UI.
+2. The product feature configurations are securely stored within Datadog.
+3. Remote-configuration enabled Datadog components in your environments securely poll, receive, and automatically apply configuration updates from Datadog. Tracing libraries that are deployed in your environments communicate with Agents to request and receive configuration updates from Datadog instead of directly polling Datadog.
+
+## Supported environments
+
+Remote Configuration works in environments where supported Datadog components are deployed. Supported Datadog components include:
+- Agents
+- Tracers (indirectly)
+- Observability Pipeline Workers
+- Private action runners and serverless container cloud services such as AWS Fargate.
+
+Remote Configuration does not support serverless container managed apps, such as AWS App Runner, Azure Container Apps, Google Cloud Run; or functions deployed with container packaging, such as AWS Lambda, Azure Functions, and Google Cloud Functions.
```

Comment:
@mellon85 is this completely accurate?

Reply:
Sounds like this is still accurate for now. But we can iterate in a follow-up PR if necessary